auditai/misc.py

```diff
@@ -65,6 +65,9 @@ def bias_test_check(labels, results, category=None, test_thresh=0.5 **kwargs):
     --------
     print statement indicating whether specific statistical tests pass or fail
     """
+    if results.max() > 0:
```

Comment:
i don't think this makes sense. you want the test_thresh to be settable by the user. this will force it to be 50 for any dataset that contains a single positive result. i would suggest `test_thresh=None` in the signature and `if test_thresh is None: test_thresh = np.median(results)` in the body (or similar)

Reply:
oh yeah of course, good catch